Source/UICollectionView+IGListKit.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <UIKit/UIKit.h>
+
+@interface UICollectionView (IGListKit)
+/**
+ A method used to configure collectionView background and disable prefetching so cells are configured as they come on screen.
```

Comment:
I should setup a linter, but I try to keep our header docs to 120 characters and then newlining. Mind breaking this into another line?

Also we can remove implementation details from the description, I think. That'll put it under 120 chars.

> Configure the UIAppearance of all UICollectionViews optimized for use with IGListKit.